app/assets/javascripts/game.js

```diff
@@ -42,7 +52,7 @@ $(function() {
         var gameId = currentUrl.substring(currentUrl.lastIndexOf('/') + 1);
         var thisCell = this;
         var pieceIdInDB = Number( droppedItem.attr('id') ) +
-                                ( (gameId - 1) * 32 )
+                                ( (gameId - 1) * 32 );
         $.ajax( {
           type: 'PATCH',
           url: '../games/' + gameId,
```

Comment:
This comment is purely based on your commit title, I haven't checked the code specifically:

For the first iteration of getting our app up and running I would leave this logic to the back-end because it is already there.  We would only need to modify things to add more feedback (like if you pick up a piece, the view changes to show all of the valid or invalid spots -- this is something we should add LATER and instead focus on MVP).  So if we are not providing a dynamic cue, the only thing that you need to do from the front-end is call `move!(x, y)` from the piece being moved, and to the x-y coordinates of the space it's attempting to be dropped.  Then do error handling when the method returns false to inform the player to make another drag-drop, or refresh the board when the method returns true.  The method will handle everything else:  moved, valid move, obstruction, capture, and eventually check and/or check-mate.

I.e. the back-end will already return false if you try to capture your own piece, so don't reinvent the wheel. :)